src/issue.ts

```diff
@@ -279,12 +279,29 @@ function getTypes(i: ApiIssue) {
 }
 
 function isPullRequest(i: ApiIssue) {
-  return !!i.PullRequest;
+  return !!i.isPr;
+}
+
+export function getPriority(p: string): number {
```

Comment:
Little confused by this piece :) I would imagine we want a `p0` coming back as `0` instead of `1`. Additionally - if there's no priority on the issue, do we want a default?  Or could we make it null|undefined?

Reply:
This is so confusing, I confused myself! This should be a bit simpler after my next commit.

tldr the [new Proto spec](https://github.com/GoogleCloudPlatform/devrel-services/pull/24/files#diff-c0752684c0c2862f8d029c8e99e9e8ccR63-R70) has the 0 value as Prioroty unknown and I confused myself and wrote a translation function for that. I'll update this to be correct (i.e. P0 => 0)